src/frontend/js/widgets/Dashboard/components/DashboardItem/Order/DashboardItemOrder.tsx

```diff
@@ -92,25 +118,32 @@ export const DashboardItemOrder = ({
         </div>
       }
     >
-      <DashboardSubItemsList
-        subItems={order.target_courses?.map((targetCourse) => (
-          <DashboardSubItem
-            title={targetCourse.title}
-            footer={
-              <DashboardItemCourseEnrolling
-                writable={writable}
-                course={targetCourse}
-                order={order}
-                activeEnrollment={CoursesHelper.findActiveCourseEnrollmentInOrder(
-                  targetCourse,
-                  order,
-                )}
-                notEnrolledUrl={getRoutePath(DashboardPaths.ORDER, { orderId: order.id })}
-              />
-            }
-          />
-        ))}
-      />
+      <>
+        <DashboardSubItemsList
+          subItems={order.target_courses?.map((targetCourse) => (
+            <DashboardSubItem
+              title={targetCourse.title}
+              footer={
+                <DashboardItemCourseEnrolling
+                  writable={writable}
+                  course={targetCourse}
+                  order={order}
+                  activeEnrollment={CoursesHelper.findActiveCourseEnrollmentInOrder(
+                    targetCourse,
+                    order,
+                  )}
+                  notEnrolledUrl={getRoutePath(DashboardPaths.ORDER, { orderId: order.id })}
+                />
+              }
+            />
+          ))}
+        />
+        {showCertificate && !!order.certificate && (
+          <div className="dashboard-item-order__certificate__container">
+            <DashboardItemOrderCertificate order={order} />
+          </div>
+        )}
```

Comment:
In fact if user has not yet earned its certificate we could display a block telling that when user will pass all courses, it will be able to download its certificate here. (Like we did in the CourseProductItem in fact).

Reply:
At the moment serialized `Order` does not contain any `certificateDefinition`, could you add it on the API?
<img width="880" alt="Capture d’écran 2023-03-21 à 17 07 45" src="https://user-images.githubusercontent.com/9628870/226669527-777fdee3-0bfa-493c-aac1-99fb45d1eaff.png">

Reply:
Why ? Because I need at least the `certificateDefinition` in order to display the certificate block without the download button.

Reply:
We found a solution, use the product instead.

Reply:
Not implemented yet ?

Reply:
It is, we pass `product` down to `DashboardItemOrderCertificate` to display the definition if the certificate is not generated yet. 

Check out `DashboardItemCertificate`, it can work with `certificate` xor `certificateDefinition` automatically.